### ISSUE TITLE
how to get the pre-trained CNN model? 

### ISSUE THREAD

**Issue by foocker** [jnhwkim/orthoad#2]
i mean a pre-trained model that was trained on other image dataset, it can detect the anomaly location on another dataset(set A) but not model the model of the  A's normal class ? 

such as trained on ImageNet, but detection the anomaly location on mvtec dataset and just using the embeddings that getting from the Pre-trained model on ImageNet  and effect on the mvtec normal dataset.

in my opinion:
1. using encoder-decoder , or metric-learning on which you will detection but the normal data, these represent embeddings can detect the anomaly.
2. using CNN-model train on other dataset(not mvtec), but detect on mvtec dataset, you should model the statistic character about the mvtec dataset's normal class, such as Multivariate Gaussian distribution model or others. this will working in understanding way. 

i will read your paper,  i read your code a few minutes and ask the above problem.

**Comment by jnhwkim** [comment]
Thanks for your interest. I hope the below answers your inquires.

1. If I correctly understand what you mean, you can exploit encoder-decoder or other metric learning. But, here, we emphasize that our simple method can significantly defeat the comparative methods in our benchmarks. 
2. Even if we do not fine-tune using the MVTecAD dataset, we model the statistical characteristics via covariance matrices (tensor).  Moreover, using the pre-trained CNN, we exploit the general visual representations, not over-fitting to a relatively small amount of examples from the MVTecAD.
